modules/rabbitmq/src/main/java/org/testcontainers/containers/RabbitMQContainer.java

```diff
@@ -0,0 +1,334 @@
+        values.forEach(command -> {
+            try {
+                ExecResult execResult = execInContainer(command.toArray(new String[0]));
+                if (execResult.getExitCode() != 0) {
+                    logger().error("Could not execute command {}: {}", command, execResult.getStderr());
+                }
+            } catch (IOException | InterruptedException e) {
+                logger().error("Could not execute command {}: {}", command, e.getMessage());
+            }
+        });
+    }
+
+    public Integer getAmqpPort() {
+        return getMappedPort(DEFAULT_AMQP_PORT);
+    }
+
+    public Integer getAmqpsPort() {
+        return getMappedPort(DEFAULT_AMQPS_PORT);
+    }
+
+    public Integer getHttpsPort() {
+        return getMappedPort(DEFAULT_HTTPS_PORT);
+    }
+
+    public Integer getHttpPort() {
+        return getMappedPort(DEFAULT_HTTP_PORT);
+    }
+
+    /**
+     * @return AMQP URL for use with AMQP clients.
+     */
+    public String getAmqpUrl() {
+        return String.format("amqp://" + getContainerIpAddress() + ":" + getMappedPort(DEFAULT_AMQP_PORT));
+    }
+
+    /**
+     * @return AMQPS URL for use with AMQPS clients.
+     */
+    public String getAmqpsUrl() {
+        return String.format("amqps://" + getContainerIpAddress() + ":" + getMappedPort(DEFAULT_AMQPS_PORT));
+    }
+
+    /**
+     * @return URL of the HTTP management endpoint.
+     */
+    public String getHttpUrl() {
+        return String.format("http://" + getContainerIpAddress() + ":" + getMappedPort(DEFAULT_HTTP_PORT));
+    }
+
+    /**
+     * @return URL of the HTTPS management endpoint.
+     */
+    public String getHttpsUrl() {
+        return String.format("https://" + getContainerIpAddress() + ":" + getMappedPort(DEFAULT_HTTPS_PORT));
+    }
+
+    /**
+     * Sets the password for the admin (default is <pre>guest</pre>)
+     *
+     * @param adminPassword The admin password.
+     * @return This container.
+     */
+    public RabbitMQContainer withAdminPassword(final String adminPassword) {
+        this.adminPassword = adminPassword;
+        return this;
+    }
+
+    public enum SslVerification {
+        VERIFY_NONE("verify_none"), VERIFY_PEER("");
+
+        SslVerification(String value) {
+            this.value = value;
+        }
+
+        private final String value;
+    }
+
+    public RabbitMQContainer withSSL(
+            final MountableFile keyFile,
+            final MountableFile certFile,
+            final MountableFile caFile,
+            SslVerification verify,
+            boolean failIfNoCert,
+            int verificationDepth) {
+
+        return withSSL(keyFile, certFile, caFile, verify, failIfNoCert)
+                .withEnv("RABBITMQ_SSL_DEPTH", String.valueOf(verificationDepth));
+    }
+
+    public RabbitMQContainer withSSL(
+            final MountableFile keyFile,
+            final MountableFile certFile,
+            final MountableFile caFile,
+            SslVerification verify,
```

Comment:
Any reason not to make this `final` too? The same for the method below.

Reply:
Done. Missed it. Could probably do it for most/all of the other methods too, but not sure it's worth it.